docs/user/file-size.md

```diff
@@ -75,3 +75,10 @@ with open("out.pdf", "wb") as f:
 
 Using this method, we have seen a reduction by 70% (from 11.8 MB to 3.5 MB)
 with a real PDF.
+
+## Removing Sources
+
+When a page is deleting (removed from page list), the source will persist(the data may be still be used somewhere else). This will reduce the page count, while not reducing the file size. 
+To not include the code, the pages should not be added using `PdfWriter.append()` selecting the good pages only
+- Issues can arise with poor pdf formatting such as when all pages are linked to the same resource, making dropping refrences useless since there is 1 source for all pages.
```

Comment:
```suggestion
- Issues can arise with poor PDF formatting such as when all pages are linked to the same resource, making dropping references useless since there is one source for all pages.
```